src/integrations/stytch.js

```diff
@@ -59,45 +59,86 @@ export function sendMagicLink(fieldVal) {
 }
 
 export function emailLogin(featheryClient) {
-  const jwt = getStytchJwt(false);
+  const jwt = getStytchJwt();
   const queryParams = new URLSearchParams(window.location.search);
   const type = queryParams.get('stytch_token_type');
   const token = queryParams.get('token');
   const stytchClient = getAuthClient();
-  // if jwt already exists, but stytch token still in URL params, then don't try
-  // to auth again as it will fail due to the info already being used to login
-  if (jwt || authSent || !stytchClient || !type || !token || !config) return;
 
+  // If there is no auth client, no config or auth has already been sent, then return early
+  if (!stytchClient || !config || authSent) return;
+
+  // If there is an existing Stytch session when a user returns to an embedded
+  // Feathery form, we need to update the auth info from Feathery's side
+  const session = stytchClient.session.getSync();
+  // Flag so that we only submitAuthInfo once
   authSent = true;
 
-  const opts = {
-    session_duration_minutes: config.metadata.session_duration
-  };
-  let authFn;
-  if (type === 'oauth') {
-    authFn = () => stytchClient.oauth.authenticate(token, opts);
-  } else if (type === 'magic_links') {
-    authFn = () => stytchClient.magicLinks.authenticate(token, opts);
-  } else {
-    return;
+  if (jwt && session) {
+    console.log('session emailLogin', session);
+    // This path to the user email only works for magic link oauth doesn't
+    // contain the email as part of the auth factor. I do have access to a few
+    // stytch IDs (will post in github comments). Not sure if you can use that instead?
+    const authEmail =
+      session.authentication_factors[0]?.email_factor?.email_address ??
```

Comment:
This path to the user email only works for magic link oauth doesn't
contain the email as part of the auth factor. I do have access to a few
stytch IDs (will post in github comments). Not sure if you can use that instead?